src/public/modules/forms/admin/directiveViews/settings-form.client.view.html

```diff
@@ -428,6 +429,39 @@
             </div>
           </div>
         </div>
+        <div class="settings-input">
+          <div class="row">
+            <div class="col-xs-9">
+              <span class="label-custom label-medium label-bottom"
+                >Enable retries</span
+              >
+              (<a
+                translate-attr="{ href: 'LINKS.WEBHOOK_RETRIES' }"
+                target="_blank"
+                >What are retries?</a
```

Comment:
should we use stronger language, given the propensity of users to muck around with webhooks?

Reply:
hopefully we can avoid the situation where a form admin were to happily enable it without considering the requirements on the HTTP status returned or considerations around idempotency